inc/saitypes.h

```diff
@@ -1196,6 +1199,54 @@ typedef struct _sai_port_snr_list_t
     sai_port_snr_values_t *list;
 } sai_port_snr_list_t;
 
+/**
+ * @brief poe port active channel (when delivering power)
+ */
+typedef enum _sai_poe_port_active_channel_id_t
+{
+    SAI_POE_PORT_ACTIVE_CHANNEL_TYPE_A = 0,
+    SAI_POE_PORT_ACTIVE_CHANNEL_TYPE_B = 1,
+    SAI_POE_PORT_ACTIVE_CHANNEL_TYPE_A_AND_B = 2
+} sai_poe_port_active_channel_id_t;
+
+/**
+ * @brief poe port signature type (when delivering power) 
+ */
+typedef enum _sai_poe_port_signature_type_id_t
+{
+    SAI_POE_PORT_SIGNATURE_TYPE_SINGLE = 0,
+    SAI_POE_PORT_SIGNATURE_TYPE_DUAL = 1
+} sai_poe_port_signature_type_id_t;
```

Comment:
please rename the typedef to sai\_poe\_port\_signature\_type\_t

Reply:
Done, thanks.